### ISSUE TITLE
Using the SRE model for other languages

### ISSUE THREAD

**Issue by sanchit-ahuja** [RicherMans/Datadriven-GPVAD#17]
Hi,
Thank you for your work on Datadriven-GPVAD. I was able to set it up and do some inferencing for my data quickly.
I wanted to know if I can use your model SRE (or any) for languages other than English. I wanted to use your model for Hindi. Or would you suggest training your model from scratch for other languages?
Also, I wanted to know if you would recommend mixing the data points for both English and Hindi and trying to train a language-agnostic model using your work.
Thanks a lot!

**Comment by RicherMans** [comment]
Hey there,
So yeah I agree with you mixing should work.

But let me state that a bit clearer how I came  by using a "Chinese" model.
I actually used the teacher ( Trained on Audioset ) to estimate labels on a Chinese dataset.
The I just ran an student on these soft labels and obtained the Chinese model.
While the paper does not provide insights into this issue, I used that specific model for Chinese speech recognition and saw a pretty large gain (up to ~10 % CER) against the SRE model.

Thus, as far as I am concerned, just use the teacher on your Hindi dataset and then retrain a student. That should be at least alright for most usecases.
